service-discovery/README.adoc

```diff
@@ -0,0 +1,134 @@
+* Access to the OpenShift Cluster
+* RHOAS CLI installed - Please follow https://github.com/bf2fc6cc711aee1a0c2a/guides/tree/main/rhoas-cli guide
+
+// Condition out QS-only content so that it doesn't appear in docs.
+// All QS anchor IDs must be in this alternate anchor ID format `[#anchor-id]` because the +ascii splitter relies on the other format `[id="anchor-id"]` to generate module files.
+ifdef::qs[]
+[#description]
+Learn how to use RHOAS Service Discovery to automatically connect Application Services to your apps running on OpenShift Kubernetes cluster in {product-long}.
+
+[#introduction]
+Welcome to the {product-long} Service Discovery quick start. In this quick start, you'll learn how to use RHOAS CLI to automatically link your Kafka Service (and other services in the future)  {product-long}.
+endif::[]
+
+[id="proc-installing-operator_{context}"]
+== Installing and verifying RHOAS Operator
+
+RHOAS Operator (Preview) creates instances of Red Hat OpenShift Application Services in your cluster. Operator connects with Application services giving you ability to embed connection details to services like OpenShift Streams for Apache Kafka and more.
+
+Red Hat OpenShift Application Services, when added to Red Hat OpenShift, 
+provide a streamlined developer experience for building, deploying, 
+and scaling cloud-native applications in open hybrid-cloud environments. 
+
+Operator will enable developers to connect with their Application Services directly in their own cluster by using
+link:https://github.com/redhat-developer/app-services-cli[RHOAS cli] and OpenShift Console UI.
+
+Operator currently enables developers to connect with their Red Hat OpenShift Streams for Apache Kafka (RHOSAK) instances.
+
+[.screencapture]
+.Relationship between RHOAS Operator OpenShift UI and RHOAS CLI
+image::rhoas-operator.png[Operator and CLI relationships]
+
+NOTE: RHOAS Operator is already installed on various environments like OpenShift Developer Sandbox, before installation process 
+
+NOTE: Installation procedure assumes that you have access to install operators in openshift-namespace on your cluster. 
+
+.Procedure
+. Please login to your OpenShift UI instance (if you do not have OpenShift instance you can use https://developers.redhat.com/developer-sandbox and skip this procedure entirely)
+. Go to "Administrator" > "Operators" > "Operator Hub"
+. Type "RHOAS" in search tab 
+. Select "RHOAS" tile
+. Leave select "All namespaces" option checked and Press install to install your operator
+. Please wait for operator to install.
+
+[id="proc-inspecting-operator_{context}"]
+== Verifying your connection to RHOAS operator
+
+RHOAS CLI "cluster" command provides the way to expose connection details for application services like Streams for Apache Kafka.
+"rhoas cluster status" command can be used to verify RHOAS operator installation.
+
+.Procedure
```

Comment:
Some guides have `.Prerequisites` but they do not show up in the renders.

Reply:
pre-requisites aren't rendered in the task for the quick start, only in card - they will show up on the customer portal.